src/main/java/org/elasticsearch/index/query/SimpleQueryStringBuilder.java

```diff
@@ -203,4 +273,164 @@ public void doXContent(XContentBuilder builder, Params params) throws IOExceptio
     protected String parserName() {
         return SimpleQueryStringParser.NAME;
```

Comment:
I think the `NAME` constant has been moved to the builder by now, you have to rebase ;)